.github/workflows/publish.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2023 Shukant Pal
+# SPDX-License-Identifier: Apache-2.0
+name: Publish to PyPI
+
+on:
+  workflow_dispatch:
+
+jobs:
+  pypi:
+    runs-on: ubuntu-latest
```

Comment:
The python bindings can be used on any platform, right? Ubuntu is just the choice for publishing, correct?

Reply:
That's right!